setup.py

```diff
@@ -21,5 +21,5 @@
       url="https://github.com/Binary-Kitchen/doorlockd/",
       author_email="ralf@binary-kitchen.de",
       packages=find_packages(),
-      requires=["Flask", "Flask-WTF", "pyserial", "python-ldap"],
+      install_requires=["Flask", "Flask-WTF", "pyserial", "python-ldap"],
```

Comment:
Why the back and forth? Better directly introduce the final state.

Reply:
I have restrained from rewriting too much history yesterday in order for @rckTom to be able to test wether the package still builds and works on arch linux with dependencies specified in setup.py; will rebase/squash soon.

Reply:
Okay, I just tried to build the arch package. It fails with the following error: https://pastebin.com/iFEjexKS

Commenting out this line fixes the issue. Have you ever seen an error like this?

Besides that, why do we need that line? In case of the arch package, all dependencies are installed via arch repositories.

Reply:
> Okay, I just tried to build the arch package. It fails with the following error: https://pastebin.com/iFEjexKS
> 
> Commenting out this line fixes the issue. Have you ever seen an error like this?
> 
> Besides that, why do we need that line? In case of the arch package, all dependencies are installed via arch repositories.

This seems weird, I need to look at other arch packages and how they handle this.

This is the usual python way of specifying dependencies, and at least debian seems to be able to get along with this (i.e. you can either install everything with pip, or with pip inside a venv, or build a debian package and it will translate those requirements into dependencies).

If I can figure out how to fix this for arch within the next few days I'll update the PR - otherwise I'll drop this change and maybe reintroduce it later.

Reply:
So it seems that https://github.com/Binary-Kitchen/doorlockd/blob/master/Makefile#L32 is the culprit. I don't see why you would want to do `--upgrade --force-reinstall` for building a Package.

If I drop those two options it works locally, is there any reason for them to be set?